validator/client/validator_propose.go

```diff
@@ -129,5 +126,10 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64) {
 		log.WithField("error", err).Error("Failed to propose block")
 		return
 	}
-	log.WithField("hash", fmt.Sprintf("%#x", blkResp.BlockHash)).Info("Proposed new beacon block")
+	log.WithFields(logrus.Fields{
+		"blockRoot": fmt.Sprintf("%#x", blkResp.BlockHash),
```

Comment:
This is confusing, is it blockRoot or blockHash? We should be consistent with naming

Reply:
block root is the correct nomenclature, hash was an old one